src/components/StorageOverview/Events/Events.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  DashboardCardTitleHelp,
+} from '../../Dashboard/DashboardCard';
+import { ClusterOverviewContextGenericConsumer } from '../../ClusterOverview/ClusterOverviewContext';
+
+export const Events = ({ Component }) => (
+  <DashboardCard>
+    <DashboardCardHeader>
+      <DashboardCardTitle>Events</DashboardCardTitle>
+      <DashboardCardTitleHelp>help for events</DashboardCardTitleHelp>
+    </DashboardCardHeader>
+    <DashboardCardBody className="kubevirt-events__body">
```

Comment:
do i need to add separate scss file for this : **className="kubevirt-events__body"**, currently i think it is taking from  ``` sass/components/ClusterOverview/events.scss1```

Reply:
![image](https://user-images.githubusercontent.com/15073590/55570287-bcd29c80-5720-11e9-8db9-c64dc1ee654b.png)